### PR TITLE
Fixed bug in front redirect servlet and added javadoc

### PR DESCRIPTION
**Pull request by eugeniosant**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

File: admin-console/src/main/java/org/entando/entando/aps/internalservlet/system/dispatcher/FrontServletActionRedirectResult.java [hunk 1]
```diff
@@ -97,7 +98,10 @@ public void execute(ActionInvocation invocation) throws Exception {
             IURLManager urlManager = (IURLManager) ApsWebApplicationUtils.getBean(SystemConstants.URL_MANAGER, request);
             Page currentPage = (Page) reqCtx.getExtraParam(SystemConstants.EXTRAPAR_CURRENT_PAGE);
             Lang currentLang = (Lang) reqCtx.getExtraParam(SystemConstants.EXTRAPAR_CURRENT_LANG);
-            String url = urlManager.createURL(currentPage, currentLang, redirectParams, false, request);
+            ConfigInterface configManager = (ConfigInterface) ApsWebApplicationUtils.getBean(SystemConstants.BASE_CONFIG_MANAGER, request);
+            String urlStype = configManager.getParam(SystemConstants.CONFIG_PARAM_BASE_URL);
+            boolean needRequest = (null != urlStype && !urlStype.equals(SystemConstants.CONFIG_PARAM_BASE_URL_RELATIVE));
+            String url = urlManager.createURL(currentPage, currentLang, redirectParams, false, (needRequest) ? request : null);
```
[the remark this answers is not in the document]
Comment (eugeniosant) [reply admin-console/src/main/java/org/entando/entando/aps/internalservlet/system/dispatcher/FrontServletActionRedirectResult.java]: Added test

File: admin-console/src/main/java/org/entando/entando/aps/internalservlet/system/dispatcher/FrontServletActionRedirectResult.java [hunk 2]
```diff
@@ -97,7 +98,10 @@ public void execute(ActionInvocation invocation) throws Exception {
             IURLManager urlManager = (IURLManager) ApsWebApplicationUtils.getBean(SystemConstants.URL_MANAGER, request);
             Page currentPage = (Page) reqCtx.getExtraParam(SystemConstants.EXTRAPAR_CURRENT_PAGE);
             Lang currentLang = (Lang) reqCtx.getExtraParam(SystemConstants.EXTRAPAR_CURRENT_LANG);
-            String url = urlManager.createURL(currentPage, currentLang, redirectParams, false, request);
+            ConfigInterface configManager = (ConfigInterface) ApsWebApplicationUtils.getBean(SystemConstants.BASE_CONFIG_MANAGER, request);
+            String urlStype = configManager.getParam(SystemConstants.CONFIG_PARAM_BASE_URL);
```
[the remark this answers is not in the document]
Comment (eugeniosant) [reply admin-console/src/main/java/org/entando/entando/aps/internalservlet/system/dispatcher/FrontServletActionRedirectResult.java]: Fixed!

**Pull request merged**